ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -356,113 +405,248 @@ public void processVRInput()
+                                                                                    {
+                                                                                       trackerFrameGraphics.put(segmentType.getSegmentName(),
+                                                                                                                new RDXReferenceFrameGraphic(FRAME_AXIS_GRAPHICS_LENGTH));
+                                                                                    }
+                                                                                    trackerFrameGraphics.get(segmentType.getSegmentName())
+                                                                                                        .setToReferenceFrame(trackerReferenceFrames.get(segmentType.getSegmentName()).getReferenceFrame());
+
+                                                                                    if (segmentType.isFootRelated())
+                                                                                    {
+                                                                                       footstepStreaming.setTrackerVelocity(segmentType.getSegmentSide(),
+                                                                                                                            new SpatialVector(ReferenceFrame.getWorldFrame(),
+                                                                                                                                              tracker.getAngularVelocity(),
+                                                                                                                                              tracker.getLinearVelocity()));
+                                                                                       footstepStreaming.setTrackerTimestamp(segmentType.getSegmentSide(), tracker.getLastPollTimeNanos());
+                                                                                    }
+
+                                                                                    desiredPose.setToZero(trackerReferenceFrames.get(segmentType.getSegmentName()).getReferenceFrame());
+                                                                                    desiredPose.changeFrame(ReferenceFrame.getWorldFrame());
+                                                                                    desiredAngularVelocity.set(tracker.getAngularVelocity());
+                                                                                    desiredLinearVelocity.set(tracker.getLinearVelocity());
+
+                                                                                    kinematicsRecorder.recordTrackerData(segmentType, trackerReferenceFrames.get(segmentType.getSegmentName()).getReferenceFrame(), desiredAngularVelocity, desiredLinearVelocity, getTrajectoryRecordFrame());
+                                                                                 });
```

Comment:
could you left indent? using the middle click to block select the space makes it easier